cmd/snapshotgc/snapshotgc.go

```diff
@@ -231,7 +231,10 @@ func getSnapshotsForRemoval(
 
 	for _, snap := range snapshots {
 		label, found := snap.GetLabels()["pac.test.appstudio.openshift.io/event-type"]
-		if found && label == "pull_request" {
+		if found && (label == "pull_request" ||
+			label == "Merge Request" ||
+			label == "Merge_Request" ||
+			label == "Note") {
```

Comment:
Note is not enough, there is more

https://github.com/openshift-pipelines/pipelines-as-code/blob/75bc53f3a67db10312611ecd92e0e231e2307600/pkg/opscomments/comments.go#L88

Reply:
I see this is not released yet. The latest release includes logic similar to the one I implemented here.
I suggest that we wait until this is released and then use the function they provide directly instead of duplicating the code.